sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePlugin.scala

```diff
@@ -20,6 +20,9 @@ object ScalaNativePlugin extends AutoPlugin {
 
     val nativeClangPP = settingKey[File]("Location of the clang++ compiler.")
 
+    val nativeExternalDependencies =
+      taskKey[Seq[String]]("Find all external dependencies")
+
```

Comment:
Lets keep the key definition `ScalaNativePluginInternal`, before we're sure that this API is fully stable.